src/util/projects.tsx

```diff
@@ -59,3 +60,24 @@ export const isProjectWithProfiles = (project?: LxdProject): boolean =>
 
 export const isProjectWithVolumes = (project?: LxdProject): boolean =>
   project?.config["features.storage.volumes"] === "true";
+
+export const getFallbackProject = (projects: LxdProject[]) => {
+  const project =
+    projects.length < 1 || projects.find((p) => p.name === "default")
+      ? "default"
+      : projects[0].name;
+
+  return project;
+};
+
+export const getDefaultProject = (projects: LxdProject[]) => {
```

Comment:
Shall we call these `getLoginProject` and the upper one `getRawLoginProject` and move them into the `loginProject.ts` file?